src/theme/basic/index.scss

```diff
@@ -13,9 +14,33 @@ $pager-primary: #757575;
 $icon-button-disabled: rgba(0,0,0,.26);
 $hidden: transparent;
 
+.q-grid-aggregation {
```

Comment:
you don't need to use specific class just apply for any tfoot, but not for floating rows